plugin-maven/CHANGES.md

```diff
@@ -3,6 +3,7 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+* Update sortpom plugin to the newest version
```

Comment:
```
## [Unreleased]
### Changes
* Bump default sortpom version to latest `3.0.0` -> `3.2.1`. ([#1675](https://github.com/diffplug/spotless/pull/1675))
```

Reply:
Use `default` if the version becomes configurable`, remove `default` if the version stays fixed.